bundle/manifests/collector_monitoring.coreos.com_v1_prometheusrule.yaml

```diff
@@ -103,6 +103,18 @@ spec:
       labels:
         service: visualization
         severity: Warning
+    - alert: DiskBufferUsage
+      annotations:
+        message: 'Collectors potentially consuming too much node disk, {{ $value }}% '
+        summary: Detected consuming too much node disk on $labels.hostname host
+      expr: "(label_replace(sum by(hostname) (vector_buffer_byte_size{component_kind='sink',
```

Comment:
The expression is not in the format as the other alerts. Does that matter?

Reply:
In what way?  I just ran "make bundle" so if the format is off then we need to understand why